proto/ddl_service.proto

```diff
@@ -88,6 +88,7 @@ message DropSinkResponse {
 message CreateMaterializedViewRequest {
   catalog.Table materialized_view = 1;
   stream_plan.StreamFragmentGraph fragment_graph = 2;
+  uint64 parallelism = 3;
```

Comment:
Is it okay to put it into `FragmentGraph`? As it only makes sense for the streaming jobs, that is, with a `graph` in the DDL request. Seems this graph is not persisted and only used for the communication.

Reply:
Make sense. Fixed.